flowzone.yml

```diff
@@ -1483,7 +1483,13 @@ jobs:
           echo "build=${matrix}">> $GITHUB_OUTPUT
 
   is_python:
-    name: Is python
+    name: Determine compatible Python versions
+    env:
+      SUPPORTED_VERSIONS: |
+        3.8
+        3.9
+        3.10
+        3.11
```

Comment:
The single source of truth.

Reply:
Supported (released) versions are selected, as per https://devguide.python.org/versions

Reply:
This makes it very easy to update the list.